src/AdoNetCore.AseClient/AseDecimal.cs

```diff
@@ -0,0 +1,564 @@
+        }
+
+        /// <summary>
+        /// Returns a value indicating if a is less than or equal to b.
+        /// </summary>
+        /// <returns>True if a is less than or equal to b.</returns>
+        public static bool operator <=(AseDecimal a, AseDecimal b)
+        {
+            return a.Backing <= b.Backing;
+        }
+
+        /// <summary>
+        /// Returns a value indicating if a is greater than or equal to b.
+        /// </summary>
+        /// <returns>True if a is greater than or equal to b.</returns>
+        public static bool operator >=(AseDecimal a, AseDecimal b)
+        {
+            return a.Backing >= b.Backing;
+        }
+
+        /// <summary>
+        /// Returns a value indicating if a is less than b.
+        /// </summary>
+        /// <returns>True if a is less than b.</returns>
+        public static bool operator <(AseDecimal a, AseDecimal b)
+        {
+            return a.Backing < b.Backing;
+        }
+
+        /// <summary>
+        /// Returns a value indicating if a is greater than b.
+        /// </summary>
+        /// <returns>True if a is greater than b.</returns>
+        public static bool operator >(AseDecimal a, AseDecimal b)
+        {
+            return a.Backing > b.Backing;
+        }
+
+        /// <summary>
+        /// Parses a string value to an AseDecimal value.
+        /// </summary>
+        /// <param name="s">The string to be parsed into an AseDecimal value.</param>
+        /// <returns>An AseDecimal structure representing the parsed string.</returns>
+        public static AseDecimal Parse(string s)
+        {
+            if (s == null)
+            {
+                return Zero;
+            }
+
+            s = s.Replace(" ", string.Empty);
+
+            if (s.Length == 0)
+            {
+                return Zero;
+            }
+
+            var isNegative = s.StartsWith("-");
+            s = s.Replace("-", string.Empty).Replace("+", string.Empty);
+            var dpIndex = s.LastIndexOf('.');
+
+            if (dpIndex >= 0)
+            {
+                s = s.TrimEnd('0');
+            }
+
+            var scale = dpIndex >= 0 && dpIndex < s.Length
+                ? s.Length - dpIndex - 1
+                : 0;
+
+            s = s.Replace(".", string.Empty);
+
+            if (BigInteger.TryParse(s, out BigInteger parsed))
+            {
+                return new AseDecimal(new BigDecimal(isNegative ? -parsed : parsed, -scale));
+            }
+
+            if (string.IsNullOrWhiteSpace(s))
+            {
+                return Zero;
+            }
+
+            throw new FormatException("The String to parse is not in the expected Format");
+        }
+
+        public static bool TryParse(string s, out AseDecimal result)
+        {
+            try
+            {
+                result = Parse(s);
+                return true;
+            }
+            catch
+            {
+                result = new AseDecimal();
```

Comment:
@senseibaka - Maybe this should be Zero?

Reply:
Will change, though the caller shouldn't be using the result if `TryParse` returns `false` anyway